doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -182,8 +182,31 @@ To bring a stalled Target node to idle state, use the ``mesh models dfu cli canc
 
 .. _bluetooth_mesh_dfu_eval_md:
 
-Composing the firmware metadata
-*******************************
+Generating the firmware metadata
+********************************
+
+Automated metadata generation
+=============================
+
+The most efficient way to generate the required DFU metadata is to use the DFU metadata extraction script integrated in the NCS build system. By enabling the ``CONFIG_BT_MESH_DFU_METADATA_ON_BUILD`` option in the
```

Comment:
Every new sentence should be on a new line and there is no limit on the line length (ncs requirement, it is opposite in zephyr).